app/services/taxonomy/parent_update.rb

```diff
@@ -0,0 +1,20 @@
+module Taxonomy
+  class ParentUpdate
+    def set_parent(content_id, parent_taxon_id: nil, associated_taxon_ids: [])
```

Comment:
Looking at the overall change, this class/method is only used from the UpdateTaxon service. Neither of these services are very big, which makes me think that having this logic in the UpdateTaxon service might be simpler, and a little less difuse. What do you think?

Reply:
Services in Ruby I believe are mostly single public method objects that do one thing (a lot of objects in content-tagger use a single 'call' method). It is not very big, but it does abstract away the way the way the parent is set.